core/App/utils/helpers.ts

```diff
@@ -49,24 +49,28 @@ export function hashToRGBA(i: number) {
   return '#' + '00000'.substring(0, 6 - colour.length) + colour
 }
 
+// DEPRECATED
```

Comment:
You could try this. It may better integrate with VSCode and others.
```
/**
 * @deprecated The method should not be used
 */
 ```

Reply:
Will update the proof request PR to remove these altogether